example/mistralai_client_dart_example.dart

```diff
@@ -2,40 +2,41 @@
 
 import 'package:mistralai_client_dart/mistralai_client_dart.dart';
 
+import 'api_key.dart';
+
 void main() async {
-  const apiKey = 'your api key here';
-  final client = MistralAIClient(apiKey: apiKey);
+  final client = MistralAIClient(apiKey: mistralApiKey);
 
   // list models
   final modelsResult = await client.listModels();
-  final models = modelsResult.data.map((e) => e.id).toList();
-  print(models.join(', '));
+  final models = modelsResult.data?.map((e) => e.id).toList();
+  print(models?.join(', '));
 
   // chat without streaming
-  final params = ChatParams(
+  const params = ChatCompletionRequest(
```

Comment:
Shouldn't it be called `request` instead of `params`? 🤔

Reply:
Haha yeah probably would be best to be consistent with the class name. Thx for catching it.

Reply:
Fixed in [e1c5263](https://github.com/nomtek/mistralai_client_dart/pull/69/commits/e1c5263ce4f704ab1e5f96036283eaefab4cdd1e)